bin/release

```diff
@@ -591,8 +542,7 @@ fi
 test "$verbose" && echo "   Running tar..." 1>&2
 ( \
     cd $tmpdir; \
-    tar cf $HDF5_VERS.tar $HDF5_VERS/Makefile \
-    `sed 's/^\.\//hdf5-'$VERS'\//' $MANIFEST` || exit 1 \
+    tar --exclude-vcs -cvvf $HDF5_VERS.tar $HDF5_VERS
```

Comment:
It looks like --exclude-vcs is a gnu extension. I'll probably switch this to use the more portable --exclude option to avoid packing the .git directory.